.github/workflows/test-react-packages.yml

```diff
@@ -0,0 +1,44 @@
+name: Meteor CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    name: Meteor react-packages tests
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Cache Meteor
+        uses: actions/cache@v4
+        with:
+          path: ~/.meteor
+          key: meteor-cache
+
+      - name: Setup Meteor
+        uses: meteorengineer/setup-meteor@v2
+        with:
+          meteor-release: '3.1.2'
```

Comment:
I suppose react-pkg runs will work with multiple Meteor versions, am I right?  

Following that, I suggest we create a Meteor version matrix to run this CI. 

``` yml
jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        meteor-version: [ '3.0.0', '3.1.0', '3.1.1', '3.1.2']
      fail-fast: false
    name: Meteor ${{ matrix.meteor-version }} react-packages tests
    steps:
      - name: Checkout repository
        uses: actions/checkout@v4
      - name: Cache Meteor
        uses: actions/cache@v4
        with:
          path: ~/.meteor
          key: meteor-${{ matrix.meteor-version }}-cache
      - name: Setup Meteor
        uses: meteorengineer/setup-meteor@v2
        with:
          meteor-release: ${{ matrix.meteor-version }}

Reply:
If we want to test this on different Meteor versions, we should also upgrade or downgrade the Meteor app used to run the tests. Upgrading is easy, downgrading less so. We can keep the test runner at the minimum Meteor version we support, making upgrades the only application on running the tests.

For now, I focused on what we actually need, and comparing tests across versions isn’t so important I think. Maybe ensuring they run in Meteor 2 is the test case, but I don’t see it as critical right now. I find it more important to ensure a matrix for React versions than for Meteor.

If this works for you, we can adjust the setup as we identify specific needs in new changes or reports.

Reply:
agree